src/index.js

```diff
@@ -123,30 +127,30 @@ function getHookScript (hookName, relativePath, cmd) {
     ])
   }
 
-  // Can't find npm message
-  var npmNotFound = '> husky - Can\'t find npm in PATH. Skipping ' + cmd + ' script in package.json'
+  // Can't find package manager message
+  var pmNotFound = '> husky - Can\'t find ' + packageManager + ' in PATH. Skipping ' + cmd + ' script in package.json'
```

Comment:
Probably. Right now it will almost always say "Can't find npm in PATH" since it only checks if yarn exists if it's already in the PATH (because we only use yarn if we detected it in PATH already).

I'll fix.